.travis.yml

```diff
@@ -9,6 +9,9 @@ notifications:
     on_success: change
     on_failure: change
 before_install:
+  # Install the latest version of Yarn (required for compatibility with `renovate`)
```

Comment:
nit: Perhaps replace the text inside the parens with something like this?

> Xenial's built-in version v1.15.2 is outdated

Reply:
Done.